source/_integrations/tplink.markdown

```diff
@@ -106,6 +106,7 @@ Devices not listed below may work but if you encounter issues submit a bug repor
 - KP125M
 - P110
 - HS100 (UK Hardware version 4.1 with firmware 1.1.0)
+- P115 (V1.0)
```

Comment:
```suggestion
- P110
- P115
- HS100 (UK Hardware version 4.1 with firmware 1.1.0)
```
Let's change the order & remove the version information.
The HS100 is a special case, as it's not a TAPO device and the only reason for this note is as they changed the protocol between hardware releases.